lib/actions/createSpreadsheet.js

```diff
@@ -1,40 +1,17 @@
-const { google } = require('googleapis');
 const { messages } = require('elasticio-node');
-const retry = require('async-retry');
 const { GoogleOauth2Client } = require('../client');
 const { QuotaManager } = require('../quotaManager');
-const { getRetriesFromConfig } = require('../common');
 
 
-// eslint-disable-next-line no-unused-vars
-async function processAction(msg, cfg, snapshot) {
+async function processAction(msg, cfg) {
+  this.logger.info('Incoming configuration: %j', cfg);
```

Comment:
It's not the best idea to log both msg and cfg. Especially on the 'info' level. Logging should be enhanced in the way it is possible to track a message

Reply:
deleted